src/css/index.scss

```diff
@@ -0,0 +1,105 @@
+section.contact {
```

Comment:
use the class is enough,  no need to add `section`. what if you need to reuse the same paddings on other elements that might not be a `section`?

Don't include anything that's not absolutely needed is the best practice.

Reply:
Please consider fix all the similar issues below. e.g. `section.contact h2`

Why don't you just create a css class for all `h2`? They looks very similar in the whole page.

Reply:
Also you need some sort of reset css to get a clean base first. Please see: https://github.com/nzittraining/cv-assignment/blob/master/src/css/global.scss#L1

Also I ran your code on my local. It is about 60% similar to design but still not reach the industry standard. I think you will need to try harder to be more detail focused to be successful on the job market. 

For example, all the margin/padding, spacing and positions are quite off compared to https://nzittraining.wixsite.com/cv-assignment. Things like colors, border styles and font family are all different from what we expect. These are the areas that you really need tp pay attention to.